docs/packages/wattpm/reference.md

```diff
@@ -0,0 +1,186 @@
+import Issues from '../../getting-started/issues.md';
+
+# Watt Commands
+
+## `create`, `init` or `add`
+
+Creates a new Watt project.
```

Comment:
Not just create, right? Please mention that those are delegated to the create-wattpm module.